crates/grpc/src/bundler.rs

```diff
@@ -49,6 +49,7 @@ impl BundlerService {
         Ok(uos)
     }
 
+    // TODO: add send bundle to flashbots
```

Comment:
add CLI mode: `./bundler --send-bundle flashbots` or `./bundler --send-bundle eth-client`

Reply:
For this, here's how I could implement it

add the `send-bundle` command in `bin/bundler/bundler`. If `flashbots`, `bundler` should use the `send_next_bundle_flashbots` method by default. There are three places I could indicate the intent to send bundle via `flashbots`
* adding an additional field to the [send_bundle](https://github.com/Vid201/aa-bundler/blob/356cee4e4c7b7aa0bee02be3280e9cc928b063df/crates/grpc/src/bundler.rs#L52) function
* adding an additional field to the [Bundler](https://github.com/Vid201/aa-bundler/blob/356cee4e4c7b7aa0bee02be3280e9cc928b063df/crates/bundler/src/bundler.rs#L13) struct;
* adding an additional field to the [bundler_service_run](https://github.com/Vid201/aa-bundler/blob/356cee4e4c7b7aa0bee02be3280e9cc928b063df/crates/grpc/src/bundler.rs#L193)

Let me know which is the best option

Reply:
I would make `SendBundleMode` enum and add an additional field to the Bundler struct. Enum is parsed from CLI and passed as an additiona field to the bundler_service_run which pushes it to the Bundler struct constructor.